homeassistant/components/tplink/strings.json

```diff
@@ -187,6 +187,9 @@
       },
       "fan_sleep_mode": {
         "name": "Fan sleep mode"
+      },
+      "pir_enabled": {
+        "name": "PIR enabled"
```

Comment:
IMO, this is correct. The goal of this PR is to control the state of the motion sensor in the switch (enabled/disabled). This is a user setting, and the user decides if PIR should be enabled or disabled for the switch. You can control it from the Kasa app, and with this PR you can also control it via HA.

Here is how it looks in the Kasa app:

<img width="301" alt="Photos_2024-11-05_06-24-46" src="https://github.com/user-attachments/assets/52877410-015b-47cd-a356-ced93a45acc7">

Instead of `PIR enabled` we can name it `Motion Sensor` for consistency. But I'm fine with any name.

Reply:
Fixed conflicts and renamed this to "Motion sensor". It is not consistent with some other `X enabled` switches, but it's more understandable than rather technical PIR so I think it's an improvement.